src/bin/ci/resources/papi_v1_application.conf

```diff
@@ -2,6 +2,8 @@ include required(classpath("application.conf"))
 include "build_application.inc.conf"
 include "papi_application.inc.conf"
 
+system.input-read-limits.lines = 512000
```

Comment:
Changes to this file only affect Pipelines API `v1` tests. `v2` is gonna have a bad time.

Update the config for both in one place by adding it to `papi_application.inc.conf` listed just above.